packages/loaders/src/Skeleton.example.md

```diff
@@ -0,0 +1,54 @@
+```jsx
+const { XXXL, XXL, MD } = require('@zendeskgarden/react-typography/src');
+const { Button } = require('@zendeskgarden/react-buttons/src');
+const { Checkbox, Label } = require('@zendeskgarden/react-checkboxes/src');
+const { zdColorKale800, zdColorWhite, zdColorBlack } = require('@zendeskgarden/css-variables');
+
+initialState = {
+  isLoading: true,
+  isDarkMode: false
+};
+
+const StyledCol = styled(Col)`
+  background-color: ${props => (props.isDarkMode ? zdColorKale800 : zdColorWhite)};
+  color: ${props => (props.isDarkMode ? zdColorWhite : zdColorBlack)};
+  padding: 18px;
+`;
+
+<Grid>
+  <Row alignItems="start">
+    <StyledCol md={8} isDarkMode={state.isDarkMode}>
+      <XXXL>
+        {state.isLoading ? <Skeleton dark={state.isDarkMode} /> : 'There. You see Lord Vader.'}
```

Comment:
How are we thinking about this strategy when the line wraps? (Shrink your browser to see.) I can understand recommending multiple `<Skeleton>` elements, but the `font-size` vs. `line-height` isn't really taken into account.

Reply:
I think this is something that we will have to provide in the design documentation/recommendations for this.